packages/easy-coding-standard/src/Console/Command/MarkdownCodeFormatterCommand.php

```diff
@@ -0,0 +1,96 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Symplify\EasyCodingStandard\Console\Command;
+
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symplify\EasyCodingStandard\Application\SingleFileProcessor;
+use Symplify\EasyCodingStandard\Configuration\Exception\NoMarkdownFileException;
+use Symplify\SmartFileSystem\SmartFileInfo;
+use Symplify\SmartFileSystem\SmartFileSystem;
+
+final class MarkdownCodeFormatterCommand extends Command
+{
+    /**
+     * @var SmartFileSystem
+     */
+    private $smartFileSystem;
+
+    /**
+     * @var SingleFileProcessor
+     */
+    private $singleFileProcessor;
+
+    public function __construct(SmartFileSystem $smartFileSystem, SingleFileProcessor $singleFileProcessor)
+    {
+        $this->smartFileSystem = $smartFileSystem;
+        $this->singleFileProcessor = $singleFileProcessor;
+
+        parent::__construct();
+    }
+
+    protected function configure(): void
+    {
+        $this->setName('markdown-code-format');
+        $this->setDescription('Format markdown code');
+        $this->addArgument('markdown-file', InputArgument::REQUIRED, 'The markdown file');
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output): int
+    {
+        /** @var string $markdownFile */
+        $markdownFile = $input->getArgument('markdown-file');
+        if (! file_exists($markdownFile)) {
+            throw new NoMarkdownFileException(sprintf('Markdown file %s not found', $markdownFile));
+        }
+
+        /** @var string $content */
+        $content = file_get_contents($markdownFile);
+        preg_match_all('#\`\`\`php\s+([^\`\`\`]+)\s+\`\`\`#', $content, $matches);
```

Comment:
Could you add link to https://regex101.com/ with example? Just so it's easier to debug later in case of troubles

Reply:
I've added regex link above preg_match_all https://regex101.com/r/4YUIu1/1

Reply:
Thank you :+1: